internal/common/conversion/misc.go

```diff
@@ -45,3 +45,11 @@ func ExpandStringListFromSetSchema(set *schema.Set) []string {
 	}
 	return res
 }
+
+func ExpandStringPointerListFromSetSchema(set *schema.Set) *[]string {
```

Comment:
not sure it's worth it to add this function when ExpandStringListFromSetSchema is available.
you can just say result := ExpandStringPointerListFromSetSchema(s) and use &result

Reply:
good point! Changing it